evenodd.c

```diff
@@ -0,0 +1,14 @@
+/*Ashank Kumar*/
```

Comment:
Great job! My only suggestion for you is to include the program description in your comment at the top